### PR TITLE
[ENHANCEMENT] Insight.io and PRs badge

### PR DESCRIPTION
**Pull request by Pl4gue**
## Description
Added two badges to README, the [PRs welcome](http://makeapullrequest.com/) and the [Insight.io](https://insight.io/github.com/Chalarangelo/30-seconds-of-code/tree/master/?source=0) badge

## What does your PR belong to?
- [ ] Website
- [ ] Snippets
- [x] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by Pl4gue** [comment]
What kind of trouble? It works for me @Chalarangelo

**Comment by Chalarangelo** [comment]
@Pl4gue as far as I can tell it works, but there's a huge link below, too. Something wrong with the markup? Preview the file to see what I mean.

File: static-parts/README-start.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![Logo](/logo.png)
 
-# 30 seconds of code [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/30-seconds-of-code/Lobby) [![Travis Build](https://travis-ci.org/Chalarangelo/30-seconds-of-code.svg?branch=master)](https://travis-ci.org/Chalarangelo/30-seconds-of-code)
+# 30 seconds of code [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/30-seconds-of-code/Lobby) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) | [![Travis Build](https://travis-ci.org/Chalarangelo/30-seconds-of-code.svg?branch=master)](https://travis-ci.org/Chalarangelo/30-seconds-of-code) [![Insight.io](https://img.shields.io/badge/insight.io-Ready-brightgreen.svg)]((https://insight.io/github.com/Chalarangelo/30-seconds-of-code/tree/master/?source=0)
```
Comment (Chalarangelo) [inline static-parts/README-start.md]: I also think that there's no need for the `|` separator between the badges, most projects have them in one line, so I suggest we do the same.

**Comment by Chalarangelo** [comment]
I agree with @iamsoorena, now that we have more badges, they should be on the line below the title.

**Pull request merged**